docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -35,7 +35,7 @@ The `error` prop toggles the error state, the `helperText` prop can then be used
 
 ## Multiline
 
-The `multiline` prop transforms the text field into a [textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) or a [TextareaAutosize](/components/textarea-autosize/).
+The `multiline` prop transforms the text field into a [Textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) or a [Textarea Autosize](/components/textarea-autosize/).
```

Comment:
I think that we should use the same name people we use the components in JSX.

```suggestion
The `multiline` prop transforms the text field into a [textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) or a [TextareaAutosize](/components/textarea-autosize/).
```

Reply:
I'm a bit confused here. The component name should be in capitals right?
Eg: 
```
The `select` prop makes the text field use the [Select](/components/selects/) component internally.
```

Reply:
I think that it should depends:

```jsx
<input />
<TextField />
```